src/ChatHeader.css

```diff
@@ -0,0 +1,48 @@
+.chatHeader__container{
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    align-items: center;
+    background-color: lightgrey;
+    border-left:1px solid darkgray;
+    border-bottom:1px solid darkgray;
+    flex:1.24;
+}
+
+#chat__info{
+    display:flex;
+    align-items: center;
+    flex:4;
+}
+
+.icon__buttons{
```

Comment:
no need of this style

Reply:
Then how to apply styling on the buttons? Alignment etc.

Reply:
Their is no need to style button here